sdk/storage/azure-storage-blob/src/main/java/com/azure/android/storage/blob/models/ContainerDeleteHeaders.java

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.storage.blob.models;
+
+import com.azure.android.core.util.DateTimeRfc1123;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
+
+import org.threeten.bp.OffsetDateTime;
+
+/**
+ * Defines headers for Delete operation.
+ */
+@JacksonXmlRootElement(localName = "Container-Delete-Headers")
+public final class ContainerDeleteHeaders {
+    /*
+     * If a client request id header is sent in the request, this header will
```

Comment:
Also mentioned in other open Storage PRs: eventually I would like us to just use either `id` or `ID`.
```suggestion
     * If a client request id header is sent in the request, this header will
```